cmf-cli/Commands/new/GrafanaCommand.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.IO.Abstractions;
+using System.Text.Json;
+using Cmf.CLI.Constants;
+using Cmf.CLI.Core;
+using Cmf.CLI.Core.Attributes;
+using Cmf.CLI.Core.Enums;
+using Cmf.CLI.Core.Objects;
+using Cmf.CLI.Utilities;
+
+namespace Cmf.CLI.Commands.New
+{
+    /// <summary>
+    /// Generates the Grafana folder structure
+    /// </summary>
+    [CmfCommand("grafana", ParentId = "new")]
```

Comment:
as you're consuming a parentId you should also specify a command Id, not only the name, as you may get duplicates in the hierarchy tree if we ever have a grafana command somewhere else